build.sbt

```diff
@@ -25,6 +25,7 @@ resolvers ++= Seq(
   "Broad Artifactory Snapshots" at "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot/")
 
 libraryDependencies ++= Seq(
+  "org.webjars" % "swagger-ui" % "2.2.5",
```

Comment:
It looks like there is a version 3.5.0 on webjars.org. Any reason not to use that?

Reply:
I'm using the same version as orch uses. The 3.x versions are a totally new interface, so I'm punting on that for now.

Reply:
Sounds good. There is a bug in the one that orch uses that has something to do with file uploads or downloads, but that's moot for Thurloe.

Reply:
I hate the 3.x interface, and it's a nontrivial upgrade